lua/harpoon/config.lua

```diff
@@ -164,7 +164,7 @@ function M.get_default_config()
                     )
 
                 Logger:log("config_default#create_list_item", name)
-
+                name = vim.fs.normalize(name)
```

Comment:
So how exactly does this fix it?

And if it does fix it, can we remove the normalize call above?

Let's just have one spot we call normalize.

Reply:
I think the problem is that on windows, path normalization cannot be done correctly for the telescope menu. This solution fixes that problem with the somewhat new vim.fs.normalize() method. However only using the normalize method breaks the marking and unmarking for some reason so I'll come up with a solution where normalization is done in one place without that issue.